addons/l10n_mx/data/account.account.tag.csv

```diff
@@ -0,0 +1,1083 @@
+"account_tag_701_06","701.06 Intereses a cargo de personas físicas nacional","Accounts","","4","Debitable Account"
+"account_tag_701_07","701.07 Intereses a cargo de personas físicas extranjero","Accounts","","4","Debitable Account"
+"account_tag_701_08","701.08 Intereses a cargo de personas morales nacional","Accounts","","4","Debitable Account"
+"account_tag_701_09","701.09 Intereses a cargo de personas morales extranjero","Accounts","","4","Debitable Account"
+"account_tag_701_10","701.10 Comisiones bancarias","Accounts","","4","Debitable Account"
+"account_tag_701_11","701.11 Otros gastos financieros","Accounts","","4","Debitable Account"
+"account_tag_702","702 Productos financieros","Accounts","","4","Creditable Account"
+"account_tag_702_01","702.01 Utilidad cambiaria","Accounts","","4","Creditable Account"
+"account_tag_702_02","702.02 Utilidad cambiaria nacional parte relacionada","Accounts","","4","Creditable Account"
+"account_tag_702_03","702.03 Utilidad cambiaria extranjero parte relacionada","Accounts","","4","Creditable Account"
+"account_tag_702_04","702.04 Intereses a favor bancarios nacional","Accounts","","4","Creditable Account"
+"account_tag_702_05","702.05 Intereses a favor bancarios extranjero","Accounts","","4","Creditable Account"
+"account_tag_702_06","702.06 Intereses a favor de personas físicas nacional","Accounts","","4","Creditable Account"
+"account_tag_702_07","702.07 Intereses a favor de personas físicas extranjero","Accounts","","4","Creditable Account"
+"account_tag_702_08","702.08 Intereses a favor de personas morales nacional","Accounts","","4","Creditable Account"
+"account_tag_702_09","702.09 Intereses a favor de personas morales extranjero","Accounts","","4","Creditable Account"
+"account_tag_702_10","702.10 Otros productos financieros","Accounts","","4","Creditable Account"
+"account_tag_703","703 Otros gastos","Accounts","","4","Debitable Account"
+"account_tag_703_01","703.01 Pérdida en venta y/o baja de terrenos","Accounts","","4","Debitable Account"
+"account_tag_703_02","703.02 Pérdida en venta y/o baja de edificios","Accounts","","4","Debitable Account"
+"account_tag_703_03","703.03 Pérdida en venta y/o baja de maquinaria y equipo","Accounts","","4","Debitable Account"
+"account_tag_703_04","703.04 Pérdida en venta y/o baja de automóviles, autobuses, camiones de carga, tractocamiones, montacargas y remolques","Accounts","","4","Debitable Account"
+"account_tag_703_05","703.05 Pérdida en venta y/o baja de mobiliario y equipo de oficina","Accounts","","4","Debitable Account"
+"account_tag_703_06","703.06 Pérdida en venta y/o baja de equipo de cómputo","Accounts","","4","Debitable Account"
+"account_tag_703_07","703.07 Pérdida en venta y/o baja de equipo de comunicación","Accounts","","4","Debitable Account"
+"account_tag_703_08","703.08 Pérdida en venta y/o baja de activos biológicos, vegetales y semovientes","Accounts","","4","Debitable Account"

```

Comment:
@qsb-odoo Why is this different than in the XML?  "Debitable Accounts" -> "D" and "Accounts" -> "accounts"?  (why the description in stead of the technical field)